src/style.css

```diff
@@ -0,0 +1,19 @@
+.row {
+  height: 100px;
+}
+
+.first {
+  background: red;
+}
+
+.second {
+  background: green;
+}
+
+.third {
+  background: blue;
+}
+
+body {
```

Comment:
move common style to the top of the file

Reply:
fixed, thanks